src/__tests__/mcp-server.test.ts

```
@@ -237,23 +259,34 @@ describe('AzureDevOpsMCPServer', () => {
         const activeItems = [createTestWorkItem({ state: 'Active' })]
         const inProgressItems = [createTestWorkItem({ state: 'In Progress' })]
 
-        mockDb.getWorkItemsByStateForUsers.mockImplementation((state: string, emails: string[]) => {
-          if (state === 'New') return Promise.resolve(newItems)
-          if (state === 'Active') return Promise.resolve(activeItems)
-          if (state === 'In Progress') return Promise.resolve(inProgressItems)
-          return Promise.resolve([])
-        })
+        mockDb.getWorkItemsByStateForUsers.mockImplementation(
+          (state: string, emails: string[]) => {
```

ESLint

[error] 'emails' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)